lisa/tests/scheduler/eas_behaviour.py

```diff
@@ -378,13 +378,15 @@ def test_slack(self, negative_slack_allowed_pct=15) -> ResultBundle:
             if bad_activations_pct > negative_slack_allowed_pct:
                 passed = False
 
-            slacks[task] = bad_activations_pct
+            bad_activations[task] = bad_activations_pct
 
         res = ResultBundle.from_bool(passed)
 
-        for task, slack in slacks.items():
-            res.add_metric("{} slack".format(task), slack, '%')
-
+        for task, bad_activations_pct in bad_activations.items():
+            res.add_metric(
+                "{} bad activations".format(task),
```

Comment:
Might as well go for 
```suggestion
                "{} delayed activations".format(task),
```

Otherwise LGTM